algorithms/src/fft/polynomial/dense.rs

```diff
@@ -90,17 +90,17 @@ impl<F: Field> DensePolynomial<F> {
     }
 
     /// Evaluates `self` at the given `point` in the field.
-    pub fn evaluate(&self, point: F) -> F {
+    pub fn evaluate(&self, point: &F) -> F {
```

Comment:
```suggestion
    pub fn evaluate(&self, point: F) -> F {
```
AFAIK `Field`s are `Copy`, which means it's preferable to pass them by value